presto-mysql/src/main/java/io/prestosql/plugin/mysql/MySqlClientModule.java

```diff
@@ -33,20 +27,6 @@
     protected void setup(Binder binder)
     {
         binder.bind(JdbcClient.class).to(MySqlClient.class).in(Scopes.SINGLETON);
-        ensureCatalogIsEmpty(buildConfigObject(BaseJdbcConfig.class).getConnectionUrl());
         configBinder(binder).bindConfig(MySqlConfig.class);
     }
-
-    private static void ensureCatalogIsEmpty(String connectionUrl)
```

Comment:
why?

Reply:
I had to figure out another way to do so as the methods used here are gone in newest driver.